Services/Security/UserOnboarder.cs

```diff
@@ -71,6 +71,7 @@ private async Task UpdateUserCoreAsync(User user, ClaimsPrincipal principal)
 
 		user.Email = principal.FindFirst(x => x.Type == ClaimConstants.EmailClaimType)?.Value;
 		user.Name = principal.FindFirst(x => x.Type == ClaimConstants.NameClaimType)?.Value;
+		user.Lastname = user.Name?.Split(' ').Last();
```

Comment:
Pro jistotu bych tam dal `.LastOrDefault()`, kdyby někdo z nějakého technického důvodu přicházel bez jména, ať to na tom nespadne.